sam_handler/handler.py

```diff
@@ -138,6 +139,10 @@ class SAM(TemplateHandler):
     """
 
     SAM_ARTIFACT_DIRECTORY = 'sam_artifacts'
+    standard_template_extensions = ['.yaml']
+    supported_template_extensions = (
+        standard_template_extensions + TemplateHandler.jinja_template_extensions
+    )
```

Comment:
`.yaml` is a template extension?

Reply:
Yeah. For SAM templates, we don't want to support `.json`, since json isn't supported by the SAM spec, and ".template" doesn't seem necessary. Thus, we're overriding the attribute from the base class and making it only support ".yaml".

Reply:
I made this more explicit.